core/src/main/scala/Commands.scala

```diff
@@ -58,7 +58,11 @@ trait MigrationFilesHandler[T] {
   }
 
   protected def writeSummary(ids: Seq[String]) {
-    val code = "object MigrationSummary {\n" + ids.map(
+    // need to sort the migration ids before writing them to
+    // the Summary file, so they are read in and processed
+    // in the correct order.
+    val sortedIds = ids.sortWith(_.toInt < _.toInt)
```

Comment:
The identifier `sortedIds` is not used in anywhere but the next line. Maybe it's a good idea to chain it in the next line (like `ids.sortWith(...).map(...)`)? What do you think?